scripts/properties2json.js

```diff
@@ -51,24 +51,36 @@ function writeFile(entries) {
   });
 }
 
-function getContentMessages(locale) {
+function readPropertiesFile(filePath) {
   return new Promise(function(resolve, reject) {
-    properties.read(path.join(process.cwd(), config.src, locale, 'messages.properties'), function(message_error, message_properties) {
+    propertiesParser.read(filePath, function(message_error, message_properties) {
       if (message_error && message_error.code !== 'ENOENT') {
         return reject(message_error);
       }
+      resolve(message_properties);
+    });
+  });
+}
 
-      properties.read(path.join(process.cwd(), config.src, locale, 'email.properties'), function(email_error, email_properties) {
-        if (email_error && email_error.code !== 'ENOENT') {
-          return reject(email_error);
-        }
-
+function getContentMessages(locale) {
+  return new Promise(function(resolve, reject) {
+    FS.listTree(path.join(process.cwd(), config.src, locale), function(path, stat) {
+      if (path.indexOf(".properties") !== -1 &&
```

Comment:
try not to rely on scoping here. Rename the function arg `path` to something else so that it is obviously apparent that it is different from `var path = require('path')`

Also, try using `path.ext()` instead, it's cleaner than doing string manipulation. Second also, what does the magical  `11` mean?